lib/membrane/rtcp/header.ex

```diff
@@ -26,18 +26,19 @@ defmodule Membrane.RTCP.Header do
          packet_type: pt
        },
        padding?: padding == 1,
-       length: (length + 1) * 4
+       length: length * 4
```

Comment:
This was done for a reason - do you really want to keep the packet byte size - 1? This makes no sense now. Either keep a raw value of the field in RTP packet or revert that change and keep a valid byte size like it was

Reply:
Here it makes the perfect sense:
- it's not stored, it's just returned
- it's needed to parse the body because the header is already parsed, so we'd have to subtract that header size in `RTCP.Packet.parse`

though I'll change that to `body_length` to make it more meaningful

Reply:
> it's not stored, it's just returned

It is stored in a struct under `length` key. The struct is then returned which makes no difference

> though I'll change that to body_length to make it more meaningful

Then it would make sense - it's clearly something different from the field in header, but I'd name it `body_size` as length is usually something that has to be calculated like in `Kernel.length`